STYLEGUIDE.md

```diff
@@ -0,0 +1,101 @@
+- Use enumerated lists with ``#. `` to list items with a clearly defined order, otherwise, use bulleted lists with ``* ``. Include a space between the bullet/number and the text. No blank line between elements, unless a sublist is present, but blank lines around the list. Sublist should have no spaces around elements. You should try to avoid more than two levels of lists if at all possible, and keep lists to between 3 and 10-20 elements; anything shorter should be written in prose, and anything longer should be simplified or broken into multiple lists/sublists.
+- Use ``note::`` for general callouts, ``warning::`` to warn users against procedures with potential serious consequences, and ``important::`` for key points that users don't want to miss. However, use these sparingly, only when clearly called for. Always make them separate blocks, with a blank line between the start of the block, like:
+```
+.. Note::
+
+   This is a note.
+```
+
+## Images/Figures
+
+- Organize images into directories following the Google drive
+- Lower case file names and extensions, separate with hyphens, 3-character extensions
+- For ease of visibility, explanatory images should be full width; expository or those with very simple subjects can be half-width, right aligned
+- Use PNG for all images, except photographs over 100-200 KB as a PNG (use moderately compressed JPEG); avoid sizes above 2000 x 2000 or below 1000 x 1000 if full width, or half of each if half width
+- Always include alt text describing the content of the image
+- No space before, one space ("|") after if full width
+
+
+## Writing (All Languages)
+
+- Follow the [OpenStack General Writing Guidelines](https://docs.openstack.org/doc-contrib-guide/writing-style/general-writing-guidelines.html)
+- Maintain an explanatory rather than expository tone if appropriate for the language (i.e. in English, "How" rather than "What"; e.g. "Spyder can do XXX." -> "To do XXX, you can...")
+- Focus on the user; this is documentation, not promotional material
+- Try to use simple language if possible and avoid buzzwords, jargon and acronyms
+- Make sure capitalization, spacing etc. of proper names and acronyms are correct and consistent per the original (or the canonical localized form is availible)
```

Comment:
* I don't understand what you mean by this.
* is available -> if available ?

Reply:
```suggestion
- Make sure capitalization, spacing etc. of proper names and acronyms are correct and consistent per the original (or the canonical localized form if available)
```

Yup, fixed.

Your English language powers have doubled since we last met xD